### ISSUE TITLE
Virus in Pyjamas

### ISSUE THREAD

**Issue by AntonyIR** [python/pythondotorg#1034]
Went to download Pyjamas 0.8.0 when Chrome and Windows Defender alerted me with a virus notice.
Link here: https://pypi.python.org/pypi/Pyjamas
Both of the download options are like this.

**Comment by berkerpeksag** [comment]
Thanks for the report, but this tracker is for issues with www.python.org only. We don't have any control over the Pyjamas project. Please report your problem to Pyjamas developers at https://github.com/pyjs/pyjs/issues

**Comment by malemburg** [comment]
I'm not sure how you managed to download Pyjamas 0.8.0 from PyPI: https://pypi.python.org/pypi/Pyjamas/0.8.0/ gives a 404.

The "current" release 0.8.1 looks fine in VirusTotal:

https://www.virustotal.com/en/url/4ca81e4d85e91c4fa8e709c8a9507c363cc1045771d4e194fc8cd52c05eb7b19/analysis/1479898441/
https://www.virustotal.com/en/file/54048cd38d49a58ebb89e7a23b45b69dc923edd9d2f79a67022175eea39f4918/analysis/1479898520/

Perhaps something is intercepting your download and adding a trojan ?!

In any case, Berker is right: this is not right place to report such issues. If related to PyPI itself, please report issues to the PyPI tracker: https://github.com/pypa/pypi-legacy/issues.

**Comment by AntonyIR** [comment]
Sorry, I figured since it happened from the python.org website that this would be the place to post it.

@malemburg the link i used is /pypi/Pyjamas not /pypi/Pyjamas/0.8.0/

The download works fine from the Pyjamas website, but in the link I posted both download options download:
pyjamas-0.8.1~+alpha1.tar.bz2 
Now if I'm correct it should be a zip file not .tar.bz2?

Anyways thanks for the help guys.
